cypress/integration/loggedOutFlows/showLoginModal.spec.js

```diff
@@ -0,0 +1,39 @@
+describe('Show log in modal', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.visit('/');
+  });
+
+  it('should show a log in modal on bookmark click', () => {
+    cy.findAllByRole('button', { name: /Save/ }).first().as('bookmarkButton');
+
+    cy.get('@bookmarkButton').click();
+
+    cy.findByTestId('modal-container').as('modal');
+    cy.get('@modal').findByText('Log in to continue').should('exist');
+    cy.get('@modal').findByLabelText('Log in').should('exist');
+    cy.get('@modal').findByLabelText('Create new account').should('exist');
+    cy.get('@modal').findByRole('button').first().should('have.focus');
+
+    cy.get('@modal').findByRole('button', { name: /Close/ }).click();
+    cy.get('@bookmarkButton').should('have.focus');
+    cy.findByTestId('modal-container').should('not.exist');
+  });
+
+  it('should show a log in modal on article reaction click', () => {
```

Comment:
Missing Test (blocking): Add a test to test showing the login modal for the unicorn reaction as well. Love that we have these tests. Just needs one more. 😎

Reply:
I didn't have a test for the bookmark click on the article page either (only on the feed), so I've added that in. Not sure what you think of grouping the reactions together in one test like that - let me know if it'd be better practice separated out 🙂

Reply:
I think one test is fine to test these as it's not a unit test. Also, there is nothing to reset to test each of these reactions.